tests/Jenkins/dirac_ci.sh

```diff
@@ -131,7 +131,7 @@ function installSite(){
 
   echo '==> Started installing'
 
-  installOptions="-t server $DEBUG "
+  installOptions="$DEBUG "
```

Comment:
why removing the server option from there ?

Reply:
`fullServer` option is added when DIRACOS is not used.

Reply:
Yes, but what happen when DIRACOS is used ?

Reply:
IIUC with DIRACOS we don't need the "-t server" switch, at least not in Jenkins where the directory structure used is simple (no "pro" nor "old" links).

Reply:
TBH I don't exactly know the impact of the `server`, `fullserver` and `client` option is, besides External package